graph.go

```diff
@@ -612,7 +614,7 @@ func (g *Graph) deleteProgeny(id TransactionID) {
 	}
 }
 
-func (g *Graph) validateTransaction(tx Transaction) error {
+func (g *Graph) validateTransaction(tx *Transaction) error {
```

Comment:
What about validating incoming transactions parent seeds? We need to ensure that parent seeds are properly validated.

Reply:
This validation would need to be done in `validateTransactionParents` in this case.

Reply:
A test must be adjusted to check for this in `graph_test.go`.